src/oas3/service.ts

```diff
@@ -38,6 +38,14 @@ export const transformOas3Service: Oas3HttpServiceTransformer = ({ document }) =
     httpService.termsOfService = document.info.termsOfService;
   }
 
+  if (document.info?.['x-logo']) {
+    httpService.logo = {
+      ...document.info['x-logo'],
```

Comment:
The `...` spread syntax is overly loose, and will allow arbitrary properties through. For forward-compatibility, I suggest doing a stricter transform like:

```suggestion
      url: document.info['x-logo'].url,
      backgroundColor: document.info['x-logo'].backgroundColor,
```

Reply:
Oh. Actually, if you really want to be safe, you should add a check that `document.info?.['x-logo']` is an object. Otherwise the code will crash if (for instance) you are editing it in a Code Editor and you make `x-logo` a boolean, e.g.
```
  x-logo: true
```

and then you'll hit `'url' is not a property of boolean` or some nonsense.